### PR TITLE
Mesh 2.2.0 release

### PR DESCRIPTION
**Pull request by lahabana**
### Description
 
Update everything to be ready to release 2.2 docs

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Review by slonka — APPROVED** [review]
I only checked https://github.com/Kong/docs.konghq.com/pull/5437/commits/b87b60437c03ea0e8113fc0ee9854eedf155295b BTW, others should've been reviewed when merging to the branch.

**Comment by slonka** [comment]
@lahabana shouldn't this update changelog and support policy as per https://github.com/Kong/kong-mesh/issues/3067#issue-1648871670 ?

**Comment by lahabana** [comment]
Not anymore since: https://github.com/Kong/docs.konghq.com/pull/5374 (Currently you haven't released the actual github release so the entries are not in version.yaml but we can do this after)

**Comment by slonka** [comment]
So we can remove that from the checklist.

**Comment by lahabana** [comment]
You can see this: https://deploy-preview-5437--kongdocs.netlify.app/mesh/2.2.x/support-policy/

I though I did already remove it from the checklist sorry!

**Comment by cloudjumpercat** [comment]
@lahabana Does this incorporate all of the changes on the `release/mesh2.2` branch? I think it does based on some of the commits I see, but wanted to double check.

**Comment by lahabana** [comment]
Yes @cloudjumpercat I've merged release-2.2 in main so everything should be there

File: app/_redirects [hunk 1]
```diff
@@ -642,6 +642,38 @@
...
+/mesh/:version/networking/ipv6/ /mesh/:version/production/dp-config/ipv6/ 
+/mesh/:version/security/secrets/  /mesh/:version/production/secure-deployment/secrets/
+/mesh/:version/security/api-access-control/ /mesh/:version/production/secure-deployment/api-access-control/
+/mesh/:version/security/api-server-auth/  /mesh/:version/production/secure-deployment/api-server-auth/
+/mesh/:version/security/dp-auth/  /mesh/:version/production/secure-deployment/dp-auth/
+/mesh/:version/security/dp-membership/  /mesh/:version/production/secure-deployment/dp-membership/
+/mesh/:version/security/certificates/ /mesh/:version/production/secure-deployment/certificates/
+/mesh/:version/explore/gui/ /mesh/:version/production/gui/
+/mesh/:version/documentation/upgrades/  /mesh/:version/production/upgrades-tuning/upgrades/
+/mesh/:version/documentation/fine-tuning/ /mesh/:version/production/upgrades-tuning/fine-tuning/
```
Comment (cloudjumpercat) [inline app/_redirects:676]:
```suggestion
/mesh/:version/documentation/fine-tuning/ /mesh/:version/production/upgrades-tuning/fine-tuning/


# Kuma IA Reference section

/docs/:version/policies/proxy-template/ /docs/:version/reference/proxy-template/


## Introduction
/docs/:version/introduction/what-is-kuma/ /docs/:version/introduction/overview-of-kuma/ 302
/docs/:version/introduction/what-is-a-service-mesh/ /docs/:version/introduction/what-is-a-service-mesh/ 302
/docs/:version/introduction/deployments/  /docs/:version/production/deployment/ 302
```

**Review by cloudjumpercat — COMMENTED** [review]
Just added a few small suggestions!

File: app/_data/kong_versions.yml [hunk 2]
```diff
@@ -346,33 +346,36 @@
...
+  version: 2.0.4
   release: 2.0.x
   releaseDate: "2022-11-04"
   endOfLifeDate: "2023-11-04"
   branch: release-2.0
 - edition: mesh
-  version: 2.1.1
+  version: 2.1.2
   release: 2.1.x
   latest: true
```
[the remark this answers is not in the document]
Comment (lahabana) [reply app/_data/kong_versions.yml]: Yes I was just waiting on the actual release to push the pointer update

[the remark this answers is not in the document]
Comment (lahabana) [reply app/_data/kong_versions.yml]: I've just added a commit with changelog and pointer update

**Pull request merged**